lua/nvim-treesitter/parsers.lua

```diff
@@ -107,6 +107,15 @@ list.astro = {
   maintainers = { "@virchau13" },
 }
 
+list.authzed = {
+  install_info = {
+    url = "https://github.com/mleonidas/tree-sitter-authzed",
+    files = { "src/parser.c" },
+    branch = "main",
```

Comment:
```suggestion
```
Not needed if it's the default branch. (You do need to add an entry to the lockfile, though.)